### PR TITLE
Fixed font, ordering and flattening of nav bar

### PR DESCRIPTION
**Pull request by SamHollings**
# Description

❓**What**: Some small fixes to make the nav bar no-long bold, to order it, and to "flatten" them so that we don't see the same page name repeated twice at the top.

🧠**Why?**: Improves the look of the site.

👨‍💻**How?**: Some small changes to the CSS and mkdocs.yml

# Checklist:
Have checked for the following:
- [x] The website still builds correctly, and you can view it using `mkdocs serve`.
- [x] There are no new "warnings" from mkdocs
- [ ] Does your page follow the [page template](https://nhsdigital.github.io/rap-community-of-practice/example_RAP_CoP_page/) (or [here in Markdown](https://github.com/NHSDigital/rap-community-of-practice/blob/main/docs/example_RAP_CoP_page.md))? (**need to make a new one specific to NHSE Data Science**)
- [ ] Spelling errors
- [ ] Consistent capitalization
- [ ] Consistent numbers
- [ ] Material features incorrectly implemented: search for code blocks and markers (e.g. !!!).
- [ ] Code snippets don't work
- [ ] Images not working
- [ ] Links not working

## Where it was tested

- Github Codespaces - 2-core, 4GB RAM, 32GB hard drive
- devcontainer.json describes further settings

### CODE REVIEW

**Comment by amaiaita** [comment]
What is data science is now at the bottom of the nav bar (i think it was better at the top as it's the basics of what we do!) - see screenshot for where it is after these changes
Screenshot 2024-01-10 at 10 35 22

i also like the subheadings in the nav bar not being bold any more as it was too much, but I think i liked the main nav bar headings being bold - but this is purely aesthetic so dont mind if you prefer it being not bold!

**Review by amaiaita — CHANGES_REQUESTED** [review]
comment added to main PR page with feedback!

File: mkdocs.yml [hunk 1]
```diff
@@ -7,7 +7,10 @@ plugins:
   - git-revision-date-localized:
       fallback_to_build_date: true
   - search
-  - awesome-pages
+  - awesome-pages:
+        order: asc
```
Comment (amaiaita) [inline mkdocs.yml:11]: think this is whats causing what is data science to be at the bottom of the nav bar and i dont think we want that

Comment (SamHollings) [reply mkdocs.yml:11]: ah yeah, so I want the case studies bit to be ordered, but I couldn't figure out how to do it.... I'll remove this and we can sort out the ordering of the case studies/projects in another ticket.

**Comment by SamHollings** [comment]
> What is data science is now at the bottom of the nav bar (i think it was better at the top as it's the basics of what we do!) - see screenshot for where it is after these changes Screenshot 2024-01-10 at 10 35 22
> 
> i also like the subheadings in the nav bar not being bold any more as it was too much, but I think i liked the main nav bar headings being bold - but this is purely aesthetic so dont mind if you prefer it being not bold!

this is resolved now! Though we will need to find another way around ordering the projects/case-studies

**Review by amaiaita — APPROVED** [review]

**Pull request merged**